docs/source/conf.py

```diff
@@ -27,6 +28,26 @@
 version = __version__
 
 
+def clean_release_notes():
+    dirname = os.path.join(os.path.dirname(__file__), 'release')
+    for rel in os.listdir(dirname):
+        for line in fileinput.input(os.path.join(dirname, rel), inplace=True):
+            if line.startswith("Announcement: napari"):
+                line = line.replace("Announcement: ", "")
+            if not line.startswith(
+                (
+                    "We're happy",
+                    'napari is a fast',
+                    "It's designed for",
+                    "images. It's built",
+                    "rendering), and ",
+                )
+            ):
+                print(line, end='')
+
+
+clean_release_notes()
```

Comment:
I wouldn't necessarily use this function... I'm pretty happy to leave all the announcement stuff in there...

Reply:
the main reason I did that was because of the title saying "Announcement".  It makes the table of contents look ridiculous (each one says announcement).  I don't mind leaving the first paragraph.  Actually, I would recommend we just remove the word "announcement" from the heading of our release template.

Reply:
Eh, I'm quite happy to keep even the announcement heading — it just makes it look like a news feed, which is fine. But I'm ok with where we are now. =)